src/external/lua/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.6)
```

Comment:
The current Ubuntu LTS (14.04) still has 2.8.12 so it would be nice if we could keep compatibility for that until the next LTS (16.04) is out.